### DISCUSSION TITLE
[Question][Hw6] z3 tutorial link is dead

### DISCUSSION THREAD

**Discussion by sjoon2455** [prosyslab-classroom/cs524-program-analysis#387] [Q&A]
Hi, 

A link to z3 tutorial(https://rise4fun.com/z3/tutorial) is not alive.

Could you please check this out?

**Comment by ShinWonho** [comment]
Hi, I hope the link below will be helpful to you.

https://github.com/Z3Prover/z3/issues/5472

**Comment by sjoon2455** [comment]
@ShinWonho Thanks!
